disttask/loaddata/scheduler.go

```diff
@@ -33,7 +35,7 @@ import (
 type ImportScheduler struct {
 	taskMeta      *TaskMeta
 	indexEngine   *backend.OpenedEngine
-	dataEngines   []*backend.OpenedEngine
+	dataEngines   sync.Map
```

Comment:
where do we have concurrently write/write or read/write? Does SplitSubtask will be called multiple times?

Reply:
Yes, will add it as soon as possible.